README.md

```diff
@@ -1,3 +1,83 @@
 # VBM Ballot Manager
 
-This tool manages the printing, mailing, and tracking of ballots to and from voters.
+This is an online tool to manage the printing, mailing, and tracking of ballots to and from voters. 
+
+This tool **does not** provide any features handled by other offline apps such as:
+- manage election ballot data
+- create ballots
+- track ballots received by jurisdiction
+- provide signature verification
+- scan/tally ballots
+
+## Workflow
+
+1. Admin authenticates into online app.
+1. Admin uploads an **Election Ballot Package** (zip file) which was exported from VxMail Ballot Creator.
+
+   Election Ballot Package contains:
+   - election.json (VotingWorks data format for election data: contests, canidates, ballot styles, precincts, etc)
+   - approved official ballots in pdf format:
+     - one for each combination of ballot style and precinct
+     - file name pattern: `ballot-style-7-precinct-8.pdf`
+
+1. Admin uploads **Voter Mailing List**
+
+   Voter Mailing List contains:
+   - id (unique identifier for voter from the jurisdiction)
+   - name
```

Comment:
These are multi-field in voter files - First Name, Middle Name, Last Name, Suffix.

Reply:
Thanks. Will update